src/XHProf.php

```diff
@@ -1,125 +0,0 @@
-<?php
```

Comment:
This one is still useful to profile with xhprof as far as I know

Reply:
Are there cases where the GLPI profiler isn't sufficient, or is it just used to profile code when not in debug mode?

Reply:
It's used to profile code when not in debug mode, among others